istio-cni/templates/configmap-cni.yaml

```diff
@@ -2,7 +2,7 @@ kind: ConfigMap
 apiVersion: v1
 metadata:
   name: istio-cni-config
-  namespace: kube-system
+  namespace: {{ .Release.Namespace }}
```

Comment:
I think it needs its own configuration variable.
When deploying on k8s ≤1.17, this namespace needs to be different from the `Release.Namespace` variable.